packages/relay/src/lib/eth.ts

```diff
@@ -498,7 +499,7 @@ export class EthImpl implements Eth {
       // retrieve the contract result details
       await this.mirrorNodeClient.getContractResultsDetails(address, contractResult.results[0].timestamp)
         .then(contractResultDetails => {
-          if (contractResultDetails?.state_changes != null) {
+          if (contractResultDetails?.state_changes != null && Array.isArray(contractResultDetails?.state_changes) && contractResultDetails?.state_changes.length > 0) {
```

Comment:
this one-liner should be a utility function somewhere, isn't it somewhere else already?  (typescript doesn't have it as a built-in?)

Reply:
no built-in function that I am aware of